src/message.rs

```diff
@@ -291,6 +291,21 @@ impl<'a, T: AsRef<[u8]> + ?Sized>
                 }
             }
 
+            RTM_NEWACTION | RTM_DELACTION | RTM_GETACTION => {
+                let err = "invalid tc action message";
+                let msg = TcActionMessage::parse(
+                    &TcActionMessageBuffer::new_checked(&buf.inner())
+                        .context(err)?,
+                )
+                .context(err)?;
```

Comment:
Is `context` required?

Reply:
good catch.  No idea how that got there.  Will fix

Reply:
It looks like I was just following an existing pattern when I did this?

If we change this pattern here we should likely change this pattern for all the other arms to remain consistent.

Thoughts?

Reply:
It's ok to keep origin pattern.
But the `context` is used to trace error path. I haven't tried what `context(err)` behaves like.

Reply:
Ok.  Then I suggest we leave it as is for now.

Reply:
The use of anyhow need to be debated in another PR  or issue. I don't like anyhow, but let's follow the precedent